stream_test.go

```diff
@@ -0,0 +1,60 @@
+package openssl
+
+import (
+	"bytes"
+	"io"
+	"os"
+	"testing"
+)
+
+func NoError(t *testing.T, err error) {
+	if err != nil {
+		t.Error(err)
+		t.FailNow()
+	}
+}
+
+func TestReader(t *testing.T) {
+	o := New()
+
+	pass := "abcd"
+	plaintext := []byte("123abc,./vvvczcekdewfeojdosndsdlsndlncnepcnodcnviorf409eofnvkdfvjfvdsoijvo")
+
+	data, err := o.EncryptBinaryBytes(pass, plaintext, BytesToKeyMD5)
+	NoError(t, err)
+
+	buf := bytes.NewBuffer(nil)
+	es := NewWriter(pass, buf, BytesToKeyMD5)
+	_, err = es.Write(plaintext)
+	NoError(t, err)
+	NoError(t, es.Flush())
+
+	t.Log(len(data), buf.Len(), bytes.Equal(data, buf.Bytes()))
+
+	_, err = io.Copy(os.Stdout, NewReader(pass, bytes.NewReader(data), BytesToKeyMD5))
+	os.Stdout.Write([]byte("\n"))
```

Comment:
Please **do not** write to `os.Stdout` but do a programmatic check of the contents. The test should be fully automated and not rely on the test execution to log something to stdout and let the user compare it.